### PR TITLE
Import Level Concept Difficulties for new 2018 levels

### PR DESCRIPTION
**Pull request by Hamms**
Based on the data contained [in this spreadsheet](https://docs.google.com/spreadsheets/d/1CgBUytWseSwiM248Ts-gLrVybRRhdP_LAUXyqcF5btM/edit#gid=0)

### CODE REVIEW

**Comment by Hamms** [comment]
@ryansloan can you spot-check the level changes to confirm that they match expectations?

**Comment by ryansloan** [comment]
I checked a handful and they LGTM

File: bin/oneoff/import_concept_difficulty_tags.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+      level = Level.find_by(name: row.fetch(:level_name))
+      unless level
+        puts "cannot find #{row.fetch(:level_name)}"
+        next
+      end
+
+      lcd = LevelConceptDifficulty.find_or_create_by(level: level)
+      lcd.update!(concept_difficulties)
+      level.write_custom_level_file
```
Comment (joshlory) [inline bin/oneoff/import_concept_difficulty_tags.rb]: Only need to do this if we have a LCD for this level, right?  Seeing a lot of edits on levels that don't actually change the LCD.

Comment (Hamms) [reply bin/oneoff/import_concept_difficulty_tags.rb]: Hmm, good catch. Lemme try and make this a bit smarter

Comment (joshlory) [reply bin/oneoff/import_concept_difficulty_tags.rb]: Probably doesn't make a huge difference — the real (long-term) work item here is to not write values to the .level file unless they're different than the default value.

Comment (Hamms) [reply bin/oneoff/import_concept_difficulty_tags.rb]: Updated!

**Pull request merged**